### PR TITLE
Feat/creating initial module

### PR DESCRIPTION
**Pull request by venkatamutyala**

### CODE REVIEW

**Review by fernandoataoldotcom — APPROVED** [review]

File: cloudflare.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+data "cloudflare_zone" "delegator" {
+  name = local.domain_to_delegate_from
+}
+
+resource "cloudflare_record" "delegation_ns_record_first" {
```
Comment (fernandoataoldotcom) [inline cloudflare.tf:6]: Is there a reason to `not for_each` these records?

**Review by fernandoataoldotcom — APPROVED** [review]
one comment, but not worth addressing to merge

File: cloudflare.tf (hunk 1 above)
Comment (venkatamutyala) [reply cloudflare.tf:6]: @fernandoataoldotcom ^^

Comment (fernandoataoldotcom) [reply cloudflare.tf:6]: rough - that's a good enough reason to leave as is...

**Pull request merged**